Research/readme.md

```diff
@@ -32,17 +49,52 @@ Once the docker image starts, the script will attempt to open your browser to th
 
 <br>
 
+## Running Jupyter Locally 
+Note: we recommend using the above approach with our Docker container, where the setup and evironment is tested and stable.
+
+Before we enable Jupyter support, follow [Lean installation](https://github.com/QuantConnect/Lean#installation-instructions)
+and [Python installation](https://github.com/QuantConnect/Lean/tree/master/Algorithm.Python#quantconnect-python-algorithm-project) to get LEAN running Python algorithms on your machine. Then be sure to build Lean at least once before the following. 
+
+**1. Installation:**
+   1. Install [JupyterLab](https://pypi.org/project/jupyterlab/):
+```
+    pip install jupyterlab
+```
+ 2.  Install [QuantConnect Python API](https://pypi.python.org/pypi/quantconnect/0.1)
+ ```
+    pip install quantconnect
+```
+ 3.  Install [pythonnet/clr-loader](https://github.com/pythonnet/clr-loader)
+ ```
+    pip install clr-loader
+```
+**2. Run Jupyter:**
+   1. Update the `config.json` file in `Lean/Launcher/bin/Debug/` folder
+ ```
+    "composer-dll-directory": ".",
```

Comment:
This requirement can be remove if the composer was a bit smarter, in research the default value is something like `microsoft.dotnet-interactive\1.0.230701\microsoft.dotnet-interactive\1.0.230701\tools\net5.0\any` vs the binaries location in backtesting,